.gitignore

```diff
@@ -5,3 +5,4 @@
 *~
 *.iml
 composer.phar
+app-prod.yaml
```

Comment:
Other possible idea. We can have `app.yaml` in the repo, also have `app.yaml` in `.gitignore`, so that it is very unlikely someone submit modified `app.yaml`. When we add the file, we can always use `-f`option.

How does it sound?